libpod/plugin/volume_api.go

```diff
@@ -129,7 +129,7 @@ func validatePlugin(newPlugin *VolumePlugin) error {
 
 // GetVolumePlugin gets a single volume plugin, with the given name, at the
 // given path.
-func GetVolumePlugin(name string, path string) (*VolumePlugin, error) {
+func GetVolumePlugin(name string, path string, timeout int) (*VolumePlugin, error) {
```

Comment:
Should volume.config be used here as well?

Reply:
it would be nice, but the import cycle is not allowed since *volumeconfig is defined in libpod but not in libpod/plugins